recipes/xapian/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "xapian" %}
+{% set version = "1.4.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://oligarchy.co.uk/xapian/{{ version }}/xapian-bindings-{{ version }}.tar.xz
+  sha256: 4519751376dc5b59893b812495e6004fd80eb4a10970829aede71a35264b4e6a
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+
+  host:
+    - python
+    - pip
+    - sphinx  # TODO: this shouldn't be needed, but there's no way to disable it in configure
+    - xapian-core {{ xapian_core }}
+    - zlib
+
+  run:
+    - python
```

Comment:
Xapian (-core) should be a run requirement

Reply:
As for zlib, `xapian-core` now defines `run_exports`, so it's not strictly necessary any more.